tests/integration/test_tracking.py

```diff
@@ -739,6 +737,7 @@ def test_partial_munmap_gap(self, tmp_path):
         peak_memory = sum(x.size for x in peak_allocations)
         assert peak_memory == 12 * PAGE_SIZE
 
+    @skip_if_macos
```

Comment:
Hm, what's the reason for this test to need to be skipped? Are we not able to reliably get adjacent pages out of mmap on Mac?

Reply:
For some reason the high water mark was higher than expected and I didn't want to fix this in this PR

Reply:
Oh, turns out it doesn't fail anymore 🤷

I will remove the skip then :)